.github/workflows/cri-evcs-stub.yml

```diff
@@ -0,0 +1,75 @@
+name: EVCS Stub Build Build & Push
```

Comment:
Might be worth renaming this file to just `evcs-stub.yml`, as it's not a CRI

Reply:
make sense - done